modules/cluster-logging-deploy-cli.adoc

```diff
@@ -342,23 +342,31 @@ spec:
         maxAge: 7d
       audit:
         maxAge: 7d
-    elasticsearch:
       nodeCount: 3 <5>
       storage:
         storageClassName: "<storage-class-name>" <6>
         size: 200G
+      resources: <7>
+        requests:
+          cpu: "1"
```

Comment:
what does `1` CPU request mean?
The default CPU request for ES is
```
	defaultESCpuRequest    = "100m"
```
https://github.com/openshift/elasticsearch-operator/blob/release-4.5/pkg/k8shandler/defaults.go#L15

Reply:
can we just omit this line?